website/static/js/profile.js

```diff
@@ -587,9 +636,36 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+
+    self.missingRequiredfield = ko.computed(function() {
+        for (var i=0; i<self.contents().length; i++) {
+            if (self.jobsOrSchools() === JOBS) {
+                if (self.contents()[i].institutionEmpty() && !(self.contents()[i].department() === '' && self.contents()[i].title() === '')) {
+                    return true;
+                }
+            }
+            else if (self.jobsOrSchools() === SCHOOLS) {
+                if (self.contents()[i].institutionEmpty() && !(self.contents()[i].department() === '' && self.contents()[i].degree() === '')) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    });
+
+    self.jobsOrSchools = ko.computed(function() {
```

Comment:
This data does not belong in the base class (`ListViewModel`)--these are the responsibility of the concrete class.